examples/armv4t/gdb.rs

```diff
@@ -73,6 +87,20 @@ impl Target for Emu {
         Ok(())
     }
 
+    fn write_register(
```

Comment:
I know you said that `read_register` would never be executed, but could you nonetheless re-add it so that it shows what the code could look like for other architectures? After all, that's sort of the point of this example, as I'm willing to bet that most consumers of the library won't actually compile / run it, just reference the code :smile: